openjdk/excludes/vendors/microsoft/ProblemList_openjdk21.txt

```diff
@@ -13,6 +13,19 @@
 #############################################################################
 
 # hotspot_all
+runtime/ErrorHandling/MachCodeFramesInErrorFile.java https://bugs.openjdk.org/browse/JDK-8313315 windows-aarch64
```

Comment:
Temurin also publishes a Windows aarch64 release for JDK21 so I'd be surprised if there's additional Msft-specific tests that need skipping for this platform? @ShelleyLambert is there a default problemlist that the Microsoft one overrides?

Reply:
@gdams, if you view the whole jdk21 problemlist file then you will see that there are other tests here that we have problemlisted in the past for either windows-aarch64 or windows-all